source/assets/sass/config/_utility.scss

```diff
@@ -0,0 +1,16 @@
+@use 'sass:map';
+@use 'sass:meta';
+@use 'scale';
+
+/// Returns all the variables defined in a module,
+/// as a map from variable names (without $)
+/// to the values of those variables.
+/// @link https://sass-lang.com/documentation/modules/meta#module-variables
+$scale: meta.module-variables('scale');
+
+// Loops through and returns all tokens from above files as CSS variables.
+@mixin tokens($map, $prefix: null) {
+  @each $tokenname, $tokenvalue in $map {
+    --#{$prefix}#{$tokenname}: #{$tokenvalue};
+  }
+}
```

Comment:
@mirisuzanne here's my approach to making these custom properties (this is included on the html element in the source/assets/sass/visual-design/_theme.scss file